cli/src/commands/workspace/add.rs

```diff
@@ -31,9 +31,34 @@ use crate::command_error::user_error;
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
+/// How to handle sparse patterns when creating a new workspace.
+#[derive(clap::ValueEnum, Clone, Debug, Eq, PartialEq)]
+enum SparseInheritance {
+    /// Copy sparse patterns from the current workspace.
+    Copy,
+    /// Do not copy sparse patterns from the current workspace, i.e. the
+    /// workspace will have a full working copy.
```

Comment:
I'd drop the "Do not copy sparse patterns... " bit. That's true for all but `Copy` after all.